.circleci/images/cuberite-circleci/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:trusty
+
+# Add LLVM 6.0 repository
+RUN apt-get update && \
+	apt-get install -y software-properties-common wget && \
+	wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add - && \
+	apt-add-repository "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-6.0 main" && \
+	apt-add-repository ppa:ubuntu-toolchain-r/test
```

Comment:
For what do you need `ubuntu-toolchain-r/test`?

Reply:
Do you have a source? I just found the deb command at https://apt.llvm.org/

Btw.: Why don't use Ubuntu Bionic (18.04LTS)? It has `clang-tidy` already in the [main repositories](https://packages.ubuntu.com/bionic/clang-tidy-6.0).

Reply:
> Do you have a source?

It's on the page you linked
```
# Also add the following for the appropriate libstdc++
deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu trusty main
```

> Why don't use Ubuntu Bionic

I'll try it out

Reply:
Ah, there it is hidden :bulb: ... Looks just soo similar